examples/jsm/controls/TrackballControls.js

```diff
@@ -49,7 +49,11 @@ class TrackballControls extends EventDispatcher {
 
 		this.keys = [ 'KeyA' /*A*/, 'KeyS' /*S*/, 'KeyD' /*D*/ ];
 
-		this.mouseButtons = { LEFT: MOUSE.ROTATE, MIDDLE: MOUSE.DOLLY, RIGHT: MOUSE.PAN };
+		this.mouseButtons = {
+			[ MOUSE.LEFT ]: STATE.ROTATE,
+			[ MOUSE.MIDDLE ]: STATE.DOLLY,
+			[ MOUSE.RIGHT ]: STATE.PAN
+		};
```

Comment:
It was originally intended to align  the API of `TrackballControls` to `OrbitControls`. Changing how `mouseButtons` is configured like in this PR interferes with this goal. It has to be:
```js
this.mouseButtons = { LEFT: MOUSE.ROTATE, MIDDLE: MOUSE.DOLLY, RIGHT: MOUSE.PAN };
```

Reply:
Wouldn't it be better to change this in OrbitControls in the same way? Currently this is a logical error (in that `switch` on lines 541-556) and this PR fixes it.